ENIGMAsystem/SHELL/Platforms/Win32/WINDOWScallback.cpp

```diff
@@ -200,9 +200,11 @@ namespace enigma
         keybdstatus[key]=1;
         if (key!=18)
         {
-          if ((lParam&(1<<29))>0)
-               last_keybdstatus[18]=keybdstatus[18], keybdstatus[18]=1;
-          else last_keybdstatus[18]=keybdstatus[18], keybdstatus[18]=0;
+          if ((lParam&(1<<29))>0) {
+            if (wParam == VK_F4) PostMessage(hWnd, WM_CLOSE, 0, 0);
```

Comment:
@JoshDreamland I could have used SendMessage instead of PostMessage but that API blocks until WM_CLOSE returns, if it even returns at all. That sounds like a bad idea right? And is it ok if I use the UPPERCASE virtual key constants here since they don't conflict with the user ones?